solr/modules/ltr/src/test/org/apache/solr/ltr/TestRerankBase.java

```diff
@@ -198,7 +199,7 @@ protected static void aftertest() throws Exception {
       jetty = null;
     }
     if (null != tmpSolrHome) {
-      FileUtils.deleteDirectory(tmpSolrHome);
+      PathUtils.deleteDirectory(tmpSolrHome.toPath());
```

Comment:
If we change tmpSolrHome from File to Path than that cleans up a bunch of other crusty bits like this too.

Reply:
addressed in b0584a3fe796ea111aa3be0b62734e3378435fb0